packaging/linux/deb/debroot.go

```diff
@@ -185,13 +185,42 @@ func Debroot(ctx context.Context, sOpt dalec.SourceOpts, spec *dalec.Spec, worke
 
 	if len(artifacts.Links) > 0 {
 		buf := bytes.NewBuffer(nil)
+		allLinksBuf := bytes.NewBuffer(nil)
```

Comment:
I'm thinking because the links file doesn't seem to support setting ownership, we probably need to move things that need ownership changes into the generic install file where we can create the link then chown in a shell script.